### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: "Browse" button is not accessible with keyboard in Push image window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1442]
**Repro Steps:**

1. Create a new dynamic web project.
2. Navigate to and select Project ->Azure->Push Image.
3. Push image dialog will open.
4. Navigate and try to access the  "Browse" button with keyboard.

**Expected Result:**
"Browse" button should be accessible with keyboard in push image dialog.

**Actual Result:**
"Browse" button is not accessible with keyboard in push image window.
**Note:** User able to access the buttons by using below keys:
Before Launch -'Alt+b'
Add- 'Alt+insert'
Remove- 'Alt+Delete'
Edit- 'Enter'
Up- 'Alt+Up'
Down- 'Alt+down'.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![buttons not accessible in push image window](https://user-images.githubusercontent.com/22956130/35728905-f9fc18ac-07c1-11e8-8646-963a5da89939.png)

**Comment by Eskibear** [comment]
Browse button in widget "com.intellij.openapi.ui.TextFieldWithBrowseButton" cannot get tab focus.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191311

**Comment by Eskibear** [comment]
Buttons in decorator cannot get tab focus either.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191322

**Comment by Eskibear** [comment]
And it's duplicated with #1440

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's a one-time configuration.
* **Quality of experience**: not impacted. A simple workaround is available. The user can input filepath of dockerfile into the text box manually. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749897
